public/old_components/scratchDatabaseExample.js

```diff
@@ -49,10 +49,60 @@ export default {
       'Rock Gnome': {desc: 'Friends with machines.'},
     },
     klasses: {
-      rogue: {desc: 'string', prestiges: {assassin: true}}
+      Barbarian: {desc: 'A fierce warrior of primitive background who can enter a battle rage.', prestiges: {'Path of the Berzerker': true, 'Path of the Totem Warrior': true}},
```

Comment:
this stuff repeated in EditClass.test.js. Factor out to an independent `.json` file that can be imported. Why do you need an `old_components` directory anyway? Can't these just be deleted?